library/tempfile.po

```diff
@@ -380,8 +380,8 @@ msgstr ""
 #: ../Doc/library/tempfile.rst:194 ../Doc/library/tempfile.rst:220
 msgid "The *dir* parameter now accepts a :term:`path-like object`."
 msgstr ""
-"Le paramètre *dir* accepte maintenant un objet fichier-compatible :term:"
-"`path-like object`."
+"Le paramètre *dir* accepte maintenant un objet simili-chemin :term:"
+"`objet simili-chemin<path-like object>`."
```

Comment:
t'as pas besoin de le recopier avant vu que le terme apparaîtra traduit :)

Reply:
Je fait quoi alors ! Le fil rouge ou le fil bleu ?? ✂️

Reply:
Tu fais comme tu le sens : si tu « forces » la traduction en écrivant :term:`français <anglais>`, tu es sûr que sur cette page c'est bien ta version qui est affichée. Sinon, tu peux laisser la traduction du terme telle qu'elle a été définie.
Ici, je pense que tu peux déléguer à la traduction « officielle ».
Sinon, j'ai noté que, dans une autre page, tu as traduit par « objet chemin ». Il faut peut-être que nous statuons sur la traduction à adopter.

Reply:
@christopheNan Je ne suis pas sûr d'avoir compris ce que tu dis. Quand tu fais ```:term:path-like object```, ça remplace tout seul par la traduction du terme ?